custom/icds_reports/views.py

```diff
@@ -855,10 +864,7 @@ def get(self, request, *args, **kwargs):
 class AWCsCoveredView(View):
     def get(self, request, *args, **kwargs):
         now = datetime.utcnow()
-        month = int(self.request.GET.get('month', now.month))
-        year = int(self.request.GET.get('year', now.year))
-        test_date = datetime(year, month, 1)
-
+        test_date = datetime(now.year, now.month, 1)
```

Comment:
Why are you removing the ability to change the date? Are these views only used for current metrics?